payment-request/allowpaymentrequest/removing-allowpaymentrequest.https.sub.html

```diff
@@ -13,26 +13,50 @@
   const path = location.pathname.substring(0, location.pathname.lastIndexOf('/') + 1);
   iframe.src = "https://{{domains[www1]}}:{{ports[https][0]}}" + path + "echo-PaymentRequest.html";
   iframe.onload = t.step_func(() => {
-    if (i === 1) {
+    if (i === 0) {
+      // 2. The iframe has now loaded (the first load).
+      // The allowpaymentrequest flag is set.
       iframe.allowPaymentRequest = false;
+      // 3. The allowpaymentrequest attribute has now been removed.
+      // The allowpaymentrequest flag is *still set* for the document.
```

Comment:
Would it be more readable if we move "set allowPaymentRequest as false" before navigating the iframe again(step 6)?

Reply:
I don't want to change the logic of what this test is doing. I'd like to revert an incorrect change, and this is already taking too much time. 😐